src/BugsnagUnity/Client.cs

```diff
@@ -353,6 +355,36 @@ void Notify(Exception[] exceptions, HandledState handledState, Middleware callba
             }
         }
 
+        private void CheckMetadaForRedactedKeys(Metadata metadata)
```

Comment:
The name `CheckMetadaForRedactedKeys` doesn't make it clear that this function will actually redact the metadata. Maybe `RedactMetadata` would be a clearer?

Reply:
agreed, thanks 👍